File: dormancymaster/src/main/java/com/dunzhixuan/dormancymaster/MainActivity.java
package com.dunzhixuan.dormancymaster;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);

		TextView textView = findViewById(R.id.txv);
		textView.setKeepScreenOn(true);

		textView.setOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View v) {
				startActivity(new Intent(MainActivity.this,SecondActivity.class));
			}
		});
	}
}

File: app/src/main/java/com/dunzhixuan/android_master/MainActivity.kt
package com.dunzhixuan.android_master

import android.os.Bundle
import android.widget.Button
import android.widget.LinearLayout
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    lateinit var container: LinearLayout;
    lateinit var btn_move:Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun onStart() {
        super.onStart()
        val ll: LinearLayout = findViewById(R.id.ll_view_layout);
        btn_move = findViewById(R.id.btn_move);
        val rightview = RightView(this)
        ll.addView(rightview)
        val scrollerView = findViewById<CustomScrollerView>(R.id.custom_scrollerView)
        btn_move.setOnClickListener {
            scrollerView.beginScroll()
        }
    }

    override fun onWindowFocusChanged(hasFocus: Boolean) {
        super.onWindowFocusChanged(hasFocus)
    }
}